helix-view/src/editor.rs

```diff
@@ -77,7 +77,8 @@ impl Default for FilePickerConfig {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+// #[serde(rename_all = "kebab-case", default)]
```

Comment:
Remove the commented line here

Reply:
:+1: 
https://github.com/helix-editor/helix/pull/798/commits/52b50785b6851055b8b7459063c344d7d5ad7931